cmd/cli/commands/devices.go

```diff
@@ -0,0 +1,92 @@
+package commands
+
+import (
+	"fmt"
+
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/sonm-io/core/proto"
+	"github.com/sonm-io/core/util"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	devicesRootCmd.AddCommand(
+		devicesLoadCmd,
+		devicesLoadRawCmd,
+	)
+}
+
+var devicesRootCmd = &cobra.Command{
+	Use:               "devices",
+	Short:             "load stored worker devices from blockchain",
+	PersistentPreRunE: loadKeyStoreWrapper,
+}
+
+var devicesLoadRawCmd = &cobra.Command{
+	Use:   "load-raw [addr]",
+	Short: "Load worker devices from blockchain",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		ctx, cancel := newTimeoutContext()
+		defer cancel()
+
+		dev, err := newDevicesClient(ctx)
+		if err != nil {
+			return fmt.Errorf("cannot create client connection: %v", err)
+		}
+
+		key, err := getDefaultKey()
```

Comment:
looks like you can use the `keystore.GetDefaultAddress()` method to obtain an eth address